ansible/roles/csm.storage.smartmon/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+#
+# MIT License
+#
+# (C) Copyright 2021-2023 Hewlett Packard Enterprise Development LP
```

Comment:
If this is a new file, the copyright header should only contain the current year

```suggestion
# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
```